socorro/unittest/cron/jobs/test_bugzilla.py

```diff
@@ -34,28 +58,15 @@ def tearDown(self):
         super(IntegrationTestBugzilla, self).tearDown()
 
     def _setup_config_manager(self, days_into_past):
-        PST = tz.gettz('PST8PDT')
-        datestring = (
-            (
-                datetime.datetime.now(PST) -
-                datetime.timedelta(days=days_into_past)
-            ).astimezone(PST).strftime('%Y-%m-%d')
-        )
-        filename = os.path.join(self.tempdir, 'sample-%s.csv' % datestring)
-        with open(filename, 'w') as f:
-            f.write('\n'.join(SAMPLE_CSV))
-
-        query = 'file://' + filename.replace(datestring, '%s')
-
         return get_config_manager_for_crontabber(
             jobs='socorro.cron.jobs.bugzilla.BugzillaCronApp|1d',
             overrides={
-                'crontabber.class-BugzillaCronApp.query': query,
                 'crontabber.class-BugzillaCronApp.days_into_past': days_into_past,
             }
         )
 
-    def test_basic_run_job(self):
+    def test_basic_run_job(self, mocker):
```

Comment:
I would have named the `mocker` argument `requests_mocker` or something like that to make it clearer what it's mocking. But it's not a big deal because these are short methods.